src/lqmining/contracts/proxyValidators.ts

```diff
@@ -36,7 +36,7 @@ export function deposit(poolId: PoolId, redeemerPk: PublicKey, expectedNumEpochs
     )
     .with_constant(5, RustModule.SigmaRust.Constant.decode_from_base16(SigmaPropConstPrefixHex + redeemerPk))
     .with_constant(7, RustModule.SigmaRust.Constant.from_byte_array(fromHex(poolId)))
-    .with_constant(13, RustModule.SigmaRust.Constant.from_i32(expectedNumEpochs))
+    .with_constant(13, RustModule.SigmaRust.Constant.from_i64(RustModule.SigmaRust.I64.from_str(Math.floor(expectedNumEpochs).toString())))
```

Comment:
Why i6?  It's i32 in specs
https://github.com/spectrum-finance/ergo-dex/blob/67a4d8d4dfb1522131a233befb37da805cba8299/contracts/lqmining/simple/Deposit.sc#L15

Reply:
i32 doesn't work. It trows error, that constant expects long type

Reply:
That only means that the deployed order is invalid

Reply:
Must be i32 according to specification

Reply:
Also, no need for `Math.floor` here, I added rounding to the method which calculates this `expectedNumEpochs`